linux/secureboot_enable_disable/check_secureboot_support_status.yml

```diff
@@ -18,12 +18,14 @@
   vars:
     skip_msg: >-
       Skip test case {{ ansible_play_name }} because secure boot is not supported
-      by {{ vm_guest_os_distribution }}.
+      by {{ vm_guest_os_distribution }} {{ guest_os_ansible_distribution_ver }}.
```

Comment:
`vm_guest_os_distribution` has already include distribution version. Please do not  change this line.

Reply:
Thanks. Done
No need to run package to test this minor changes.